pkg/controller/status/status.go

```diff
@@ -402,8 +402,8 @@ func (m *statusManager) ClearDegraded() {
 	m.lock.Lock()
 	defer m.lock.Unlock()
 	m.degraded = false
-	m.explicitDegradedReason = ""
-	m.explicitDegradedMsg = ""
+	m.explicitDegradedReason = string(operator.NotApplicable)
```

Comment:
I think this file can have its own constants, and it is probably better not to rely on any variables from the "operator" import.

Reply:
Done